packages/app-store/_utils/useIsAppEnabled.ts

```diff
@@ -0,0 +1,56 @@
+import { useState, useEffect } from "react";
+
+import type { EventTypeAppCardApp } from "../types";
+
+const useIsAppEnabled = (app: EventTypeAppCardApp, getAppData: unknown, setAppData: unknown) => {
```

Comment:
I managed to fix some of the type errors here (Might have knock on effects not sure) not blocking by any means 
```suggestion
import type { SetAppDataGeneric, GetAppDataGeneric } from "../EventTypeAppContext";
function useIsAppEnabled<TAppData extends ZodType>(
  app: EventTypeAppCardApp,
  getAppData: GetAppDataGeneric<TAppData>,
  setAppData: SetAppDataGeneric<TAppData>
){```

Reply:
Where would `TAppData` come from? I'm thinking do we generate a schema file for each event type app (ex. `apps.keys-schemas.generated.ts` and pass it into `TAppData`?

Reply:
As discussed in DMs TAppData is infered from usage on getAppData and setAppData